client/src/App.jsx

```diff
@@ -1,20 +1,43 @@
-import React from 'react';
+import React, {useState} from 'react';
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { ThemeProvider } from '@mui/material/styles';
+import theme from './theme.jsx';
 
 //Components
 import Navbar from './components/layout/Navbar';
+import Heading from './components/layout/Heading';
+import AuthOutlet from './components/routing/AuthOutlet';
+import NoAuthOutlet from './components/routing/NoAuthOutlet';
 import ReportsPage from './components/pages/ReportsPage';
 import SettingsPage from './components/pages/SettingsPage';
+import LandingPage from './components/pages/LandingPage';
+import ForgotPassword from './components/pages/ForgotPassword';
+import AuthSwitch from './components/dev/AuthSwitch.jsx';
 
 function App() {
+  const [auth, setAuth] = useState(false);
+  const handleAuth = (event) =>{
+    setAuth(event.target.checked);
+  }
+
   return (
-    <Router>
-      <Routes>
-        <Route path="/" element={<ReportsPage />} />
-        <Route path="/settings" element={<SettingsPage />} />
-      </Routes>
-      <Navbar />
-    </Router>
+    <ThemeProvider theme={theme}>
+      <Router>
+        {!auth && <Heading />}
+        <AuthSwitch checked={auth} onChange={handleAuth}/>
```

Comment:
Lol is this just a switch in the app to "log in"?

Reply:
Yea lol, I got tired of changing the auth value to get to certain pages